.circleci/config.yml

```diff
@@ -97,8 +97,7 @@ jobs:
           command: |
             if [ -n "${npm_TOKEN}" ] && [ "${CIRCLE_BRANCH}" == "master" ]; then
               echo "//registry.npmjs.org/:_authToken=${npm_TOKEN}" > ~/project/.npmrc
-              npm version prerelease --preid="${CIRCLE_BUILD_NUM}.${CIRCLE_BRANCH}.`git rev-parse --short HEAD`" --no-git-tag-version
-              npm publish --tag current
+              yarn lerna publish --canary --preid "${CIRCLE_BUILD_NUM}.${CIRCLE_BRANCH}" --yes
```

Comment:
Don't you want to add `--skip-git`?

Reply:
```
Successfully published:
 - stardust-ui-test-react@2.0.1-alpha.2+1f3c4b76
```

It will not create git tag or a version commit.